tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Monitoring NServiceBus solutions: Getting started"
+reviewed: 2017-11-07
+summary: Setting up the Particular Service Platform to monitor an NServiceBus system.
+component: Core
+versions: '[5,)'
+---
+
+This tutorial will guide you through the steps required to set up your NServiceBus solution for monitoring with the Particular Service Platform.
+
+
+## Component overview
+
+```mermaid
+graph TD
+
+subgraph Endpoint
+  nsb(NServiceBus) --> metrics
+  metrics(NServiceBus.Metrics) --> plugin
+
+  plugin(NServiceBus.Metrics.ServiceControl)
+end
+
+plugin --> q(Monitoring Queue)
+
+nsb --> auditQ(Audit Queue)
+nsb --> errorQ(Error Queue)
+
+subgraph ServiceControl
+  scinstance
+  scmonitoring
+end
+
+auditQ --> scinstance(ServiceControl Instance)
+errorQ --> scinstance
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+scinstance --> pulse
+
+```
+
+![the above renders like this](component-overview.png)
```

Comment:
That's awesome for reviewing (thanks!), but I'd remove that pic before merging to avoid getting out of sync. We can always use the online generator to validate. Are you OK with this?

Reply:
:+1: This was the intent. Glad it help with the review :)